nautobot/ipam/models.py

```diff
@@ -679,19 +676,23 @@ def get_first_available_ip(self):
     def get_utilization(self):
         """Get the child prefix size and parent size.
 
-        For Prefixes with a status of "container", get the number child prefixes. For all others, count child IP addresses.
+        For Prefixes with a type of "container", get the number child prefixes. For all others, count child IP addresses.
 
         Returns:
             UtilizationData (namedtuple): (numerator, denominator)
         """
-        if self.status == Prefix.STATUS_CONTAINER:
+        if self.type == choices.PrefixTypeChoices.TYPE_CONTAINER:
             queryset = Prefix.objects.net_contained(self.prefix).filter(vrf=self.vrf)
             child_prefixes = netaddr.IPSet([p.prefix for p in queryset])
             return UtilizationData(numerator=child_prefixes.size, denominator=self.prefix.size)
 
         else:
             prefix_size = self.prefix.size
-            if self.prefix.version == 4 and self.prefix.prefixlen < 31 and not self.is_pool:
+            if (
+                self.prefix.version == 4
+                and self.prefix.prefixlen < 31
+                and self.type != choices.PrefixTypeChoices.TYPE_POOL
+            ):
```

Comment:
Similarly, when it comes to chained `and` matches, I find `all()` to be more readable:

```suggestion
            if all([
                self.prefix.version == 4,
                self.prefix.prefixlen < 31,
                self.type != choices.PrefixTypeChoices.TYPE_POOL,
            ]):
```